Model/GastatsRaw.php

```diff
@@ -351,16 +351,34 @@ public function processGAStatsAll($start=null, $stop=null) {
 
 		echo "Pulling the Raw generic stats\n";
 		$this->processGAStats($start, $stop);
+
 		echo "Pulling the Ad stats\n";
+        $GastatsAd->loadGA();
+        $GastatsAd->GoogleAnalytics->reAuth();
```

Comment:
Can we address this spacing to be inline with the current style?

Reply:
I have gone ahead and done this. We might want to convert these to PSR-2 compliant spaced indents in the future, but I've made the new additions tabbed for the sake of matching the rest of the file.